frontend-react/src/pages/deliveries/Table/DeliveriesTable.tsx

```diff
@@ -13,10 +13,17 @@ import Spinner from "../../../components/Spinner";
 import { getReportAndDownload } from "./ReportsUtils";
 import ServicesDropdown from "./ServicesDropdown";
 
+enum DeliveriesDataAttr {
+    REPORT_ID = "reportId",
+    BATCH_READY = "batchReadyAt",
+    EXPIRES = "expires",
+    ITEM_COUNT = "reportItemCount",
+    FILE_TYPE = "fileType",
+}
 /** @todo: page size default set to 10 once paginated */
 const filterManagerDefaults: FilterManagerDefaults = {
     sortDefaults: {
-        column: "sent",
+        column: DeliveriesDataAttr.BATCH_READY,
```

Comment:
Is this the actual bug? "sent" vs "batchReadyAt"?

Reply:
Yes, because we rely on the key in the response object, and that changed from sent to batchReadyAt when we swapped over from Reports to Deliveries

Reply:
Great, just making sure.